### ISSUE TITLE
how LSTM+Glove+IA encode articles?

### ISSUE THREAD

**Issue by HAWLYQ** [alasdairtran/transform-and-tell#2]
@alasdairtran Hi, I have read your newly published paper. I'm curious about how LSTM+Glove+IA encodes articles? Does it encode each article at the sentence level or the word level?

**Comment by alasdairtran** [comment]
I encoded each article at the word level. It's simply the average of the glove embeddings of the words in the article.

**Comment by HAWLYQ** [comment]
> I encoded each article at the word level. It's simply the average of the glove embeddings of the words in the article.

so there is only an article vector for the article and there is no sentence-level attention (proposed in goodness paper) in it, right?

**Comment by alasdairtran** [comment]
Yep! Just one article vector and no attention. And we were able to (slightly) beat the ones reported Goodnews.

**Comment by HAWLYQ** [comment]
> Yep! Just one article vector and no attention. And we were able to (slightly) beat the ones reported Goodnews.

Thanks! Wonderful work! And I have one more question: Are the embeddings for byte-pair tokens randomly initialized during decoding?

**Comment by alasdairtran** [comment]
Yes in the decoder, the embeddings of the byte-pair tokens are initialised from a gaussian with mean 0 and variance 1/embed_size

**Comment by HAWLYQ** [comment]
Thanks!

**Comment by HAWLYQ** [comment]
Thanks!

**Comment by HAWLYQ** [comment]
Thanks!

**Comment by yuan93427** [comment]
@alasdairtran 
Thanks for your work!
I want to generate captions on the NYTimes800k test set , but it take me a long time to train.
Can you provide the weights `best.th `?

**Comment by alasdairtran** [comment]
Send me an email (see README file) to request the full data and model weights.
